docs/en/observability/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,99 @@
+:aws: AWS
+
+[[aws-elastic-serverless-forwarder]]
+= Elastic Serverless Forwarder
+
+// should we expand titleabbrev to AWS or Amazon Web Services Elastic Serverless Forwarder? What is correct nomenclature?
+
+++++
+<titleabbrev>AWS Elastic Serverless Forwarder</titleabbrev>
+++++
+:keywords: serverless
+:description: The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda function that ships logs from your {aws} environment to Elastic.
+
+The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda function that ships logs from your {aws} environment to Elastic.
+
+[discrete]
+[[aws-serverless-forwarder-overview]]
+= Overview
+The Elastic Serverless Forwarder can forward {aws} data to self-managed or cloud-hosted Elastic environments. It supports the following inputs:
+
+- Amazon S3 (via SQS (Simple Queue Service) event notifications)
+- Amazon Kinesis Data Streams
+- Amazon CloudWatch Logs subscription filters
+- Amazon SQS message payload
+
+[role="screenshot"]
+image::aws-serverless-lambda-flow.png[AWS Lambda flow]
+
+The Lambda deployment sets up an SQS _Continuing queue_ automatically. This is used to trigger a new function invocation that Lambda uses to continue from exactly where the last function run was terminated. By default a Lambda function runs for a maximum of 15 minutes. When processing event data there’s a possibility that the function may be exited by {aws} in the middle of processing. The function handles this scenario gracefully by keeping track of the last offset processed.
+
+Any exception or fail scenarios related to ingestion are handled by Lambda gracefully using a _Replay queue_. Data in the replay queue is stored as individual events. Lambda keeps track of any failed events and writes them to a replay queue that can then be consumed by adding an additional SQS trigger via Lambda.
+
+You can use the <<sample-s3-config-file,config.yaml>> file to configure the service for each input type, including information such as SQS queue ARN and {es} connection details. You can create multiple input sections within the configuration file to map different inputs to specific log types.
+
+The Lambda function also supports writing directly to an index, alias, or custom data stream. This enables existing {es} users to re-use index templates, ingest pipelines, or dashboards that are already created and connected to business processes.
```

Comment:
I don't really know what "are connected to business processes" means. Does that just mean they are already in use? 

```suggestion
The Lambda function also supports writing directly to an index, alias, or custom data stream. This enables existing {es} users to re-use index templates, ingest pipelines, or dashboards that already exist and are connected to business processes.
```

Reply:
Yes but I agree some examples or details would help here... @ravikesarwani could you help with this one?